runtime/handler.go

```diff
@@ -11,6 +12,11 @@ import (
 	"golang.org/x/net/context"
 )
 
+var (
+	// The indent to use for JSON responses.
+	ResponseIndent = ""
```

Comment:
@shurcooL the `SetResponseIndent` could easily do things like strip invalid indentation characters, particularly via a whitelist of simply `[" ","\t"]`, for example. Getter/Setter pattern hides implementation details and can help protect invalid usage of exported variables in a package.